tailwind.config.js

```diff
@@ -1,27 +1,42 @@
-const colors = require('tailwindcss/colors')
-
 module.exports = {
     content: [
-        './resources/views/**/*.blade.php',
-        './resources/css/**/*.css',
-        './resources/js/**/*.vue',
+        "./resources/views/**/*.blade.php",
+        "./resources/css/**/*.css",
+        "./resources/js/**/*.vue",
 
-        './vendor/rapidez/**/*.blade.php',
-        './vendor/rapidez/**/*.css',
-        './vendor/rapidez/**/*.vue',
+        "./vendor/rapidez/**/*.blade.php",
+        "./vendor/rapidez/**/*.css",
+        "./vendor/rapidez/**/*.vue",
 
-        './config/rapidez.php',
-        './vendor/rapidez/core/config/rapidez.php',
-        './vendor/rapidez/menu/config/menu.php',
-        './vendor/laravel/framework/src/Illuminate/Pagination/resources/views/tailwind.blade.php',
+        "./config/rapidez.php",
+        "./vendor/rapidez/core/config/rapidez.php",
+        "./vendor/rapidez/menu/config/menu.php",
+        "./vendor/laravel/framework/src/Illuminate/Pagination/resources/views/tailwind.blade.php",
     ],
     theme: {
         extend: {
             colors: {
-                primary: colors.green[700],
-                secondary: colors.gray[400],
+                primary: {
+                    DEFAULT: "rgb(var(--primary) / <alpha-value>)", // Theme color
+                    text: "rgb(var(--primary-text) / <alpha-value>)", // Text color that goes onto primary color
+                },
+                secondary: {
+                    DEFAULT: "rgb(var(--secondary) / <alpha-value>)", // Conversion color
+                    text: "rgb(var(--secondary-text) / <alpha-value>)", // Text color that goes onto secondary color
+                },
+                neutral: "rgb(var(--neutral) / <alpha-value>)", // Default text color
+                inactive: "rgb(var(--inactive) / <alpha-value>)", // Inactive text color
+                highlight: "rgb(var(--highlight) / <alpha-value>)", // Background highlight color
+                border: "rgb(var(--border) / <alpha-value>)", // Border color
+            },
+            borderColor: {
+                DEFAULT: "rgb(var(--border) / <alpha-value>)", // Border color default so it gets used when only using border
```

Comment:
was this necessary? why couldn't this be normal hexxes again?

Reply:
Since rapidez already has multi site theming support and we want to keep it that way